frappe/website/web_template/section_with_cards/section_with_cards.html

```diff
@@ -11,6 +11,12 @@
 }) -%}
 <div class="mt-6 {{ col_classes }}">
 	<div class="card {{ card_classes }} h-100">
+	    {%- if image -%}
+	    {{ frappe.render_template('templates/includes/image_with_blur.html', {
```

Comment:
Wrong indentation?

Reply:
I will fix this I don't know how this happened as in my text editor it was correct.